Documentation/Functions/If.rst

```diff
@@ -20,6 +20,9 @@ There is no else property available. The "else" branch of an "if" statement is a
 missing feature. You can implement a workaround by a logic based on the
 :ref:`stdwrap-override-conditions`.
 
+Simple "if empty use different value" conditions can be built with the
```

Comment:
```suggestion
Simple "if empty use different value" conditions can be build with the
```

Reply:
"can be built" is correct english.
https://textranch.com/c/can-be-built-or-can-be-build/